api/src/test/java/uk/gov/hmcts/payment/api/componenttests/util/PaymentsDataUtil.java

```diff
@@ -114,6 +114,7 @@ public Payment populateCardPaymentToDb(String number) throws Exception {
             .paymentMethod(PaymentMethod.paymentMethodWith().name("card").build())
             .paymentProvider(PaymentProvider.paymentProviderWith().name("gov pay").build())
             .paymentStatus(PaymentStatus.paymentStatusWith().name("success").build())
+            .customerReference("customerRef" + number)
```

Comment:
is this a slight change in spec?

Reply:
this came from Dave's PR, https://github.com/hmcts/ccpay-payment-app/pull/1696/files#diff-8ddae71591aac46e0ba744a252a2d38675f19c5a7a0abd3b76e7af2f0e545a63R117.